component/src/main/java/com/synopsys/integration/alert/component/authentication/descriptor/AuthenticationUIConfig.java

```diff
@@ -46,6 +46,7 @@
     private static final String LABEL_LDAP_GROUP_SEARCH_FILTER = "LDAP Group Search Filter";
     private static final String LABEL_LDAP_GROUP_ROLE_ATTRIBUTE = "LDAP Group Role Attribute";
     private static final String LABEL_SAML_ENABLED = "SAML Enabled";
+    private static final String LABEL_SAML_WANT_ASSERTIONS_SIGNED = "Sign Assertions";
```

Comment:
Are the changes to this class even needed at this point?

Reply:
Not necessarily, but I added them for now while they still exist just in case